ui/job-view/headerbars/WatchedRepo.jsx

```diff
@@ -119,65 +120,65 @@ export default class WatchedRepo extends React.Component {
       );
     }
     return (
-      <span className="btn-group">
+      <UncontrolledDropdown>
         <a
           href={changeRepoUrl}
           className={`watched-repo-main-btn btn btn-sm ${btnClass} ${activeClass}`}
-          type="button"
           title={status}
         >
           <i className={`fa ${icon} ${pulseIcon} ${color}`} /> {watchedRepo}
         </a>
-        <button
-          className={`watched-repo-info-btn btn btn-sm btn-view-nav ${activeClass}`}
-          type="button"
-          title={`${watchedRepo} info`}
-          aria-label={`${watchedRepo} info`}
-          data-toggle="dropdown"
-        ><span className="fa fa-info-circle" /></button>
-        {watchedRepo !== repoName && <button
-          className={`watched-repo-unwatch-btn btn btn-sm btn-view-nav ${activeClass}`}
-          onClick={() => unwatchRepo(watchedRepo)}
-          title={`Unwatch ${watchedRepo}`}
-        ><span className="fa fa-times" /></button>}
-
-        <ul className="dropdown-menu" role="menu">
+        <DropdownToggle
+          tag="span"
+        >
+          <button
+            className={`watched-repo-info-btn btn btn-sm btn-view-nav ${btnClass} ${activeClass}`}
+            type="button"
+            title={`${watchedRepo} info`}
+            aria-label={`${watchedRepo} info`}
+          ><span className="fa fa-info-circle" /></button>
+          {watchedRepo !== repoName && <button
+            className={`watched-repo-unwatch-btn btn btn-sm btn-view-nav ${btnClass} ${activeClass}`}
+            onClick={() => unwatchRepo(watchedRepo)}
+          ><span className="fa fa-times" /></button>}
+        </DropdownToggle>
+        <DropdownMenu right className={'watched-repo-menu'}>
           {status === 'unsupported' && <React.Fragment>
-            <li className="watched-repo-dropdown-item">
+            <DropdownItem header>
               <span>{watchedRepo} is not listed on <a
```

Comment:
We don't need this surrounding ``<span>`` anymore.  Not sure if we ever did, tbh.  :)